backend/tracim_backend/config.py

```diff
@@ -1272,14 +1272,6 @@ def _check_ldap_config_validity(self):
             self.check_mandatory_param(
                 "LDAP_URL", self.LDAP_URL, when_str="when ldap is in available auth method",
             )
-            self.check_mandatory_param(
-                "LDAP_BIND_DN", self.LDAP_BIND_DN, when_str="when ldap is in available auth method",
-            )
-            self.check_mandatory_param(
-                "LDAP_BIND_PASS",
-                self.LDAP_BIND_PASS,
-                when_str="when ldap is in available auth method",
-            )
```

Comment:
@inkhey can you think it is possible to add parameter in development.ini "allow_anonymous_ldap_bind=True" to fix this ?

I think its a good idea to explain explicitly if allow anonymous or not.

Reply:
Yes, i do think that's a good idea. but i do think we should probably be more strict with something like "ldap_bind_anonymous" boolean instead of a parameter that have different behavior. if we set to anonymous, the auth params should not been used.

 @yalh76, thanks for this contributions, are you at ease doing this change ?

The important things to do normal when adding a new parameters are:
- updating the `development.ini.sample` documentation
- add the parameter to config.py with potential extra check.
- update the https://github.com/tracim/tracim/blob/develop/backend/doc/env_settings.md documentations.

If you are not at ease doing this, i propose you to continue adapting tracim to make all things works with Yunohost and let us doing the "software polishing" (you can give us write access to you PR).

Reply:
I'm not at ease to do that, I have no knowledge of python :/ Just able to do some copy/paste. 
So yes, it's better you do the software polishing and I do the YunoHost adaptation ^^

Reply:
@yalh76 I added a config parameter `ldap_bind_anonymous`, you must set it as true for tracim to allow anonymous auth. I tested it and it seems to work as expected, can you validate this behavior work for you too ?
As soon as you validate this, we will be able to merge this change.